MVP Module/Source/Feed Presentation/FeedPresenter.swift

```
@@ -7,6 +7,7 @@ import FeedFeature
 
 protocol FeedLoadingView {
 	func display(_ viewModel: FeedLoadingViewModel)
+    func display(error: String)
```

This method could pass an optinal error string: `func display(error: String?)`, so the Presenter can decide when to show/hide the message.

For example, in the `didStartLoadingFeed()` method, the Presenter can call  `loadingView.display(error: .none)` to hide the error.

This way, the ViewController doesn't need to decide when to hide the error. It's much easier to manage presentation state when all changes are centralized in the presenter.